.github/CHANGELOG.md

```diff
@@ -31,6 +32,19 @@
   plt.hist(samples, bins=100)
   plt.show()
   ```
+
+* Adds the `sf.ops.GKP` operation, which allows the Gottesman-Kitaev-Preskill
+  state to be initialized on both the Bosonic and Fock backends.
+  [(#553)](https://github.com/XanaduAI/strawberryfields/pull/553)
+  [(#546)](https://github.com/XanaduAI/strawberryfields/pull/546)
```

Comment:
I didn't add a code example here, since the code example directly above shows off the GKP op!